packages/material-ui/src/ListItemAvatar/ListItemAvatar.js

```diff
@@ -4,55 +4,39 @@ import clsx from 'clsx';
 import withStyles from '../styles/withStyles';
 import ListContext from '../List/ListContext';
 
-export const styles = theme => ({
+export const styles = {
   /* Styles applied to the root element. */
   root: {
-    width: 36,
-    height: 36,
-    fontSize: theme.typography.pxToRem(18),
-    marginRight: 4,
+    minWidth: 56,
   },
-  /* Styles applied to the root element when. */
+  /* Styles applied to the root element when context.alignItems="flex-start". */
```

Comment:
Context is an implementation detail. What does this actually look like for a user?

Reply:
I will update the description. We can try:
```suggestion
  /* Styles applied to the root element when <ListItem alignItems="flex-start" />. */
```

Reply:
```suggestion
  /* Styles applied to the root element when the parent `ListItem` uses `alignItems="flex-start"`. */
```
?